go-controller/pkg/kubevirt/pod.go

```diff
@@ -343,3 +343,9 @@ func ZoneContainsPodSubnetOrUntracked(watchFactory *factory.WatchFactory, lsMana
 	// to a node
 	return hostSubnets, !util.IsContainedInAnyCIDR(annotation.IPs[0], hostSubnets...), nil
 }
+
+// IsPodOwnByVirtualMachine returns true if the pod is own by a
+// kubevirt virtual machine, false otherwise.
+func IsPodOwnByVirtualMachine(pod *corev1.Pod) bool {
```

Comment:
```suggestion
// IsPodOwnedByVirtualMachine returns true if the pod is owned by a
// kubevirt virtual machine, false otherwise.
func IsPodOwnedByVirtualMachine(pod *corev1.Pod) bool {
```